chapters/rum/_toctree.yml

```
@@ -26,3 +26,23 @@
   - local: chapter1/10
     title: Quiz de final de capitol
     quiz: 1
+
+- title: 2. Using 🤗 Transformers
```

```suggestion
- title: 2. Folosirea 🤗 Transformers
```
Just a suggestion since I don't know if titles need to be translated